### DISCUSSION TITLE
Externel monitor brightness maxed out at 50%

### DISCUSSION THREAD

**Discussion by Wood-Fruit** [MonitorControl/MonitorControl#814] [Q&A]
Monitor: LG 27UL500
Version: 4.0.2 6595
Description: The monitor reaches max brightness when the adjustment bar provided by the app is at 50%.

EDIT: searched in the discussion, found the solution at #745, sorry about the repost.

**Comment by krotuk** [comment]
On MacBook Pro (13-inch, 2017, Two Thunderbolt 3 ports) Monitor LG 34WK95U-W connected through Thunderbolt 3 cable from the LG monitor

hello I tried MonitorControl I have the same problem. I solved it by setting the brightness on the monitor through the joystick of the monitor to the highest possible id est 100 and setting the "Combined dimming switchover point" to the far right. Then you can enjoy controlling the brightness almost perfectly through your keyboard.
But, there is a but, doing this I observed that when I set the brightness down to under 50% luminosity through MonitorControl this time, the monitor gets incredible hot when I touch it at the back after a while. As if the monitor actually "never quits 100%" luminosity (which it does not because I set it up there and disabled "Auto Brightness" on the LG monitor).

**Reply by Wood-Fruit** [reply]
I'm using the same computer, A1708. Display Port didn't work but HDMI works perfectly. so I suspect the displayport is on DP 1.2 or something that doesn't support hardware communication. I suggest using an HDMI cable and see what happens.
